Gemfile

```
@@ -20,6 +21,9 @@ group :test do
   gem 'rspec_junit_formatter', '~> 0.6.0', require: false
   gem 'simplecov-cobertura', '~> 3.0.0', require: false
   gem 'webmock', '~> 3.25', require: false
+  gem 'tzinfo'
```

RuboCop (1.82.1)

[convention] Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `tzinfo` should appear before `webmock`.

(Bundler/OrderedGems)